pypif/obj/common/property.py

```diff
@@ -67,16 +71,32 @@ def conditions(self):
 
     @property
     def method(self):
-        return self._method
+        warn("method has been deprecated in favor of methods")
+        return self._methods
 
     @method.setter
     def method(self, method):
+        warn("method has been deprecated in favor of methods")
         self._validate_type('method', method, dict, Method)
-        self._method = self._get_object(Method, method)
+        self._methods = self._get_object(Method, method)
```

Comment:
Instead of this, `self.methods = method` so that it calls the setter, which might change as some point.